### PR TITLE
Create API

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,152 @@
...
+
+```go
+username := "username"
+password := "password"
+client := api.NewAPIClient("https://listmonk.3mdeb.com", &username, &password)
+```
+
+The URL should point to the Listmonk service (in our case it's
+https://listmonk.3mdeb.com). The username and password should match an
+administrator account.
```
Comment (artur-rs) [inline README.md:32]: @PLangowski are there any problems with user access management in listmonk? We shouldn't use single account creds used by multiple employees

Comment (PLangowski) [reply README.md:32]: I haven't tested using other accounts.

Comment (PLangowski) [reply README.md:32]: It seems that multiple accounts are not yet supported in listmonk https://github.com/knadh/listmonk/issues/543

File: api/client.go [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+
+	if err != nil {
+		return nil, err
+	}
+
+	var inc_campaign *listmonk.Campaign = nil
+	var incCampaignLaunchDate string
+	for _, camp := range campaigns {
+		if camp.Name == campaign.Name+"_inc" {
+			inc_campaign = camp
```
Comment (dancios) [inline api/client.go:132]: I see camelCase vs _ in code:)

File: api/client.go [hunk 3]
```diff
@@ -0,0 +1,459 @@
...
+
+	listmonk "github.com/Exayn/go-listmonk"
+)
+
+type APIClient struct {
+	BaseURL        string
+	Username       *string
+	Password       *string
+	Client         *listmonk.Client
+	MailingListIDs map[string]uint
```
Comment (dancios) [inline api/client.go:19]: What if few instances woud work try access map

**Review by dancios — CHANGES_REQUESTED** [review]
i probably forgot to click "review chages" after writing comments :D

**Review by artur-rs — APPROVED** [review]

**Pull request merged**